Justfile

```diff
@@ -150,5 +149,10 @@ check-protocol-schema:
     env {{protocol_schema_env}} cargo test -p protocol-schema-check --profile dev-artifacts
     env {{protocol_schema_env}} cargo run -p protocol-schema-check --profile dev-artifacts
 
-check_build_public_libraries:
-    cargo check {{public_libraries}}
+publishable := "cargo metadata --format-version 1 | jq -r '.packages[] | select(.name | startswith(\"near-\")) | select(.publish == null or (.publish | length > 0)) | .name'"
+check-publishable-separately:
+    #!/usr/bin/env bash
+    for pkg in $({{publishable}}); do
+        echo "Checking $pkg..."
+        cargo check -p $pkg
```

Comment:
Would be worthwhile to also check for:

* `--no-default-features`
* `--all-features`

for each package. I suggest adding a way to pass through arbitrary flags similar to what's seen in `check-cargo-clippy`.